__init__.py

```diff
@@ -14,19 +14,20 @@
 import imp
 import platform
 
-from retriever.lib.compile import compile_script
+from retriever.lib.compile import compile_json
 from retriever._version import __version__
 
 current_platform = platform.system().lower()
 if current_platform != 'windows':
     import pwd
 
 VERSION = __version__
-MASTER = True
+MASTER = False
 COPYRIGHT = "Copyright (C) 2011-2016 Weecology University of Florida"
 REPO_URL = "https://raw.github.com/weecology/retriever/"
 MASTER_BRANCH = REPO_URL + "master/"
-REPOSITORY = MASTER_BRANCH if MASTER else REPO_URL + VERSION + "/"
+JSON_BRANCH = REPO_URL + "scripts-to-json/"
```

Comment:
A couple of questions about this:

1. With the current approach is there a reason not to use the `data-package` branch instead? I noticed that the scripts in the two branches aren't identical.
2. Is there anything that would stop us from merging the JSON files first so that we don't have to mess around with changing this and then changing it back? My intuition (and a quick test) suggests that we can add all of the JSON scripts to the scripts directory without any influence on the current version of the retriever. So, we could go ahead and merge in the JSON scripts separately now and then change these lines back so that they are still hitting `master`.